src/scripts/main.js

```diff
@@ -2,6 +2,40 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const parentElem = document.body;
+  const newDiv = document.createElement('div');
+  const titleElem = document.createElement('h2');
+  const descriptionELem = document.createElement('p');
```

Comment:
There is a typo in the variable name `descriptionELem`. It should be `descriptionElem` to maintain consistency with the naming convention used for `titleElem`.